nixos/modules/virtualisation/kvmgt.nix

```diff
@@ -0,0 +1,58 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.virtualisation.kvmgt;
+  kernelPackages = config.boot.kernelPackages;
+  vgpuOptions = {
+    uuid = mkOption {
+      type = types.string;
+      description = "UUID of VGPU device.";
+    };
+  };
+in {
+  options = {
+    virtualisation.kvmgt = {
+      enable = mkEnableOption "KVMGT (iGVT-g) VGPU support.";
```

Comment:
While technically correct, maybe add some info what this is about. Like, "allows virtual guests to share host's graphics card. Currently only one graphical device can be shared"